app/uk/gov/hmrc/excisemovementcontrolsystemapi/utils/DateTimeService.scala

```diff
@@ -18,19 +18,26 @@ package uk.gov.hmrc.excisemovementcontrolsystemapi.utils
 
 import com.google.inject.{ImplementedBy, Singleton}
 
-import java.time.Instant
-import java.time.temporal.ChronoUnit
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, ZoneOffset, ZonedDateTime}
 
 @ImplementedBy(classOf[DateTimeServiceImpl])
 trait DateTimeService {
   def timestamp(): Instant
-
-  def timestampToMilliseconds(): Instant
 }
 
 @Singleton
 class DateTimeServiceImpl extends DateTimeService {
-  override def timestamp(): Instant = Instant.now()
+  override def timestamp(): Instant  = Instant.now()
+}
+
+object DateTimeService {
+  implicit class DateTimeFormat(val dateTime: Instant) extends AnyVal {
 
-  override def timestampToMilliseconds(): Instant = timestamp().truncatedTo(ChronoUnit.MILLIS)
+    implicit def asStringInMilliseconds: String = {
```

Comment:
used this because the instant.toString is no very good when working transforming to a string. This will exclude milliseconds if these are 0: For example:

Instant.parse(2024-04-01T10:20:12.120Z) toString -> "2024-04-01T10:20:12.12Z"
Instant.parse(2024-04-01T10:20:12.100Z) toString -> "2024-04-01T10:20:12.1Z"
Instant.parse(2024-04-01T10:20:12.00000Z) toString -> "2024-04-01T10:20:12Z"